explorer/Explorer.tsx

```diff
@@ -196,6 +187,11 @@ export class Explorer
         )
     }
 
+    private persistedQueryParamsByGrapher: Map<
```

Comment:
oooh I like this idea. if grapher switching ends up being a long term thing then doing this right and minimizing url noise could be worth it. is there a way we can add tests? this code has been terrible to work with (my fault) and if we could get tests in there could be good.

Reply:
yes, tests are included on the new PR to be merged with Next!

Reply:
jk turns out I can change the target of this PR